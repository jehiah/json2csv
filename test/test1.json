{"a": 1, "b": "asdf\n"}
{"a" : null}
{"a": "\"quote test\""}
{"a": "regular quote"}
