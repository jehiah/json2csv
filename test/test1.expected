1

"""quote test"""
"regular quote"
